code/frameworks/sveltekit/src/mocks/app/state.svelte.ts

```diff
@@ -0,0 +1,129 @@
+/**
+ * Inspired by the the code:
+ * {@link https://github.com/sveltejs/kit/blob/main/packages/kit/src/runtime/client/state.svelte.js}
+ *
+ * The differences:
+ *
+ * - Legacy Svelte support is not included
+ * - Not using classes (internal coding style preference)
+ *
+ * @module
+ */
+
+/* eslint-disable prefer-const */
+import { fn } from 'storybook/test';
+
+/**
+ * Copied from:
+ * {@link https://github.com/sveltejs/kit/blob/7bb41aa4263b057a8912f4cdd35db03755d37342/packages/kit/types/index.d.ts#L2552-L2581}
+ * There were build issues after adding `@sveltejs/kit` as `devDependency`: Error: namespace child
+ * (hoisting) not supported yet
+ */
+// eslint-disable-next-line @typescript-eslint/no-namespace
+declare namespace App {
+  export interface Error {
+    message: string;
+  }
+
+  export interface Locals {}
+
+  export interface PageData {}
+
+  export interface PageState {}
+
+  export interface Platform {}
+}
+
+/**
+ * Copied from:
+ * {@link https://github.com/sveltejs/kit/blob/7bb41aa4263b057a8912f4cdd35db03755d37342/packages/kit/types/index.d.ts#L1102-L1143}
+ */
+export interface Page<
+  Params extends Record<string, string> = Record<string, string>,
+  RouteId extends string | null = string | null,
+> {
+  url: URL;
+  params: Params;
+  route: {
+    id: RouteId;
+  };
+  status: number;
+  error: App.Error | null;
+  data: App.PageData & Record<string, any>;
+  state: App.PageState;
+  form: any;
+}
```

Comment:
```suggestion
```

Given that no-one imports from this file directly - it's always used as an alias - I don't think there's any value to exporting the types.